### ISSUE TITLE
Hex

### ISSUE THREAD

**Issue by Lafriakh** [colorjs/color-space#35]
is there any plan to add hex, to convert from rgb or hsl.
`hex.hsl`
`hex.rgb`

**Comment by dfcreative** [comment]
hex is actually rgb color space. Prob you want to see [color-parse](https://github.com/colorjs/color-parse) or [color-stringify](https://github.com/colorjs/color-stringify)

**Comment by edwardhorsford** [comment]
I came here looking for this too. As I see it there's two ways of representing rgb colour space - they're equivalent. It would be great to support both as I suspect many users will be needing to work with one of them.

**Comment by dy** [comment]
Can you elaborate use-case?

**Comment by edwardhorsford** [comment]
@dy I should preface to say this library was really useful, and did what I needed 👍 

---

My use case was for two things:
* I had an array of `hsl` values that I needed in `hex` for display purposes. I'd assumed a library that converts colours would support hex.
* I then had some `hex` values that I wanted to convert to `hsl` to work on them.

Following your link to `color-parse` above, this is what I ended up using:
```
let rgb = rgbToHex(round(space.lab.rgb(pantone.records[colour].lab))).toUpperCase()
let hsl = space.rgb.hsl(colorParse(`#${colour}`).values)
```
Where `rgbToHex` was a function I grabbed from stack overflow.

Possibly I've done these in a cumbersome way - but if not it would have been really helpful if this could have been built in. To do the conversion has needed to require `round` as well as the `color-parse` library.

If you don't want to include, a note in the readme would have been helpful. I was confused and thought I was misunderstanding things because it seemed intuitive to me that a colour parsing library would support one of the more common colour formats.
